pyugrid/ugrid.py

```diff
@@ -599,11 +688,8 @@ def save_as_netcdf(self, filepath):
         """
         mesh_name = self.mesh_name
 
-        # FIXME: Why not use netCDF4.Dataset instead of renaming?
-        from netCDF4 import Dataset as ncDataset
         # Create a new netcdf file.
-        with ncDataset(filepath, mode="w", clobber=True) as nclocal:
-
+        with Dataset(filepath, mode='w', clobber=True) as nclocal:
```

Comment:
we should probably have a way for the user to pass in options to netCDF4 - netcdf version, etc....